curriculum/challenges/english/25-front-end-development/lecture-working-with-css-transforms-overflow-and-filters/672bccae6e556cd81cef6af2.md

```diff
@@ -10,6 +10,30 @@ dashedName: what-is-margin-collapsing
 
 Watch the lecture video and answer the questions below.
 
+# --transcript--
+
+What is margin collapsing, and how does it work?
+
+Margin collapsing is a fundamental concept in CSS that often confuses newcomers to web development.
+
+This behavior occurs when the vertical margins of adjacent elements overlap, resulting in a single margin equal to the larger of the two.
+
+Understanding margin collapsing is important for precise control over spacing and layout in web design. So, let's get into how margin collapsing works and explore some common scenarios where it occurs.
+
+In CSS when two vertical margins come into contact with each other they'll collapse, this means that instead of adding together the larger margin wins and determines the space between the elements. This behavior applies only to vertical margins top and bottom and not horizontal margins, not the left and right. So, here's an example to illustrate this concept: `Code example on screen to show the concept`.
```

Comment:
not sure how I feel about all of these `Code example on screen to show the concept` and `visual example` on screen comments. 

It doesn't look like we are using that convention elsewhere in the other transcripts.

maybe we should just explain what it will look like instead.